howto/monitor/cos.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
```

Comment:
I think we need to differentiate two parts here:

1. General machine monitoring with grafana-agent
2. Individual workload monitoring

As we can not provide 2) for an official release of Anbox Cloud yet, maybe we focus on 1) only?